sale_timesheet_rounded/README.rst

```diff
@@ -0,0 +1,126 @@
+======================
+Sale Timesheet Rounded
+======================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Ftimesheet-lightgray.png?logo=github
+    :target: https://github.com/OCA/timesheet/tree/12.0/sale_timesheet_rounded
```

Comment:
```suggestion
    :target: https://github.com/OCA/timesheet/tree/13.0/sale_timesheet_rounded
```

Reply:
Readme regenerated by bot no need to touch it manually